_about/PULL_REQUESTS.md

```diff
@@ -0,0 +1,44 @@
+# Pull Request Submission and Lifecycle
+
+We appreciate direct contributions to the provider codebase. Here's what to
+expect:
+
+ * For pull requests that follow the guidelines, we will proceed to reviewing
+  and merging, following the provider team's review schedule. There may be some
+  internal or community discussion needed before we can complete this.
+ * Pull requests that don't follow the guidelines will be commented with what
+  they're missing. The person who submits the pull request or another community
+  member will need to address those requests before they move forward.
+
+## Pull Request Lifecycle
+
```

Comment:
Maybe step 1 would be to open a github issue announcing your intention to create this new feature. Kind of like a feature proposal. That gives the community a place to discuss the approach and use cases while the contributor begins work on the PR.

It could also generate some enthusiasm and get community members ready to review the upcoming PR. We would be able to give early feedback on the proposed approach before too much work goes into the PR.

I think I'm ok with it either way though. It can be some unnecessary overhead to create an issue if the feature you're building is small.

Reply:
I like this idea - it's what we're doing in the kubernetes-alpha respository already, so I can't think of a reason not to do it here as well.

Reply:
Added

Reply:
Should we also add a recommendation to open a new issue to discuss intentions for new features / changes if one is not already available? 
The way I read the paragraph below, the contributor is encouraged to only discuss if there is already an open issue. I might just as well have misunderstood it.

Reply:
Good thought - I think I had it in there in one revision and took it out. I'll add some language there.